CovidTracking/CovidTracking/Scene/Tracking/TrackingViewController.swift

```diff
@@ -8,18 +8,167 @@
+    
+    let bag = DisposeBag()
+    
+    var dataSource: RxTableViewSectionedAnimatedDataSource<TrackingSectionModel>!
+    var viewModel: TrackingViewModel!
+    
+    private var addBarButton: UIBarButtonItem!
+    private var editBarButton: UIBarButtonItem!
+    
     override func viewDidLoad() {
         super.viewDidLoad()
         
         configureViews()
+        configureNavigationBar()
+        configureTableView()
+        
+        bindView()
+    }
+    
+    private func bindView() {
+        viewModel = TrackingViewModel(details: [])
+        bindViewModel()
     }
     
     private func configureViews() {
         title = L10n.trackingTitle.localized()
         
         view.backgroundColor = .systemPink
     }
+    
+    private func configureNavigationBar() {
+        editBarButton = UIBarButtonItem(barButtonSystemItem: .edit,
+                                         target: self, action: nil)
+        addBarButton = UIBarButtonItem(barButtonSystemItem: .add,
+                                         target: self, action: nil)
+        
+        navigationItem.do {
+            $0.rightBarButtonItem = addBarButton
+            $0.rightBarButtonItem?.tintColor = .systemPink
+            $0.leftBarButtonItem = editBarButton
+            $0.leftBarButtonItem?.tintColor = .systemPink
+        }
+    }
+    
+    private func configureTableView() {
+        registerCell()
+        configureDataSource()
+    }
+    
+    private func registerCell() {
+        tableView.do {
+            $0.register(cellType: TrackingTableViewCell.self)
+            $0.rowHeight = 220
+            $0.separatorColor = .clear
+        }
+    }
+    
+    private func configureDataSource() {
+        dataSource = RxTableViewSectionedAnimatedDataSource<TrackingSectionModel>(
+            animationConfiguration: AnimationConfiguration(insertAnimation: .right,
+                                                           reloadAnimation: .none,
+                                                           deleteAnimation: .left),
+            configureCell: configureCell,
+            canEditRowAtIndexPath: canEditRowAtIndexPath,
+            canMoveRowAtIndexPath: canMoveRowAtIndexPath)
+    }
+}
+
+extension TrackingViewController {
+    private var configureCell: RxTableViewSectionedAnimatedDataSource<TrackingSectionModel>.ConfigureCell {
+        return { _, tableView, indexPath, detail in
+            var cell = tableView.dequeueReusableCell(for: indexPath, cellType: TrackingTableViewCell.self)
+            cell.bind(to: TrackingCellViewModel(detail: detail))
+            return cell
+        }
+    }
+    
+    private var canEditRowAtIndexPath: RxTableViewSectionedAnimatedDataSource<TrackingSectionModel>.CanEditRowAtIndexPath {
+        return { [unowned self] _, _ in
+            if self.tableView.isEditing {
+                return true
+            } else {
+                return false
+            }
```

Comment:
```suggestion
            return tableView.isEditing
```
kiểm tra lại xem closure này có tham số nào là tableView không nhé

Reply:
vâng ạ